docs: fixed some typo and grammatical errors in README.MD (#1776)
webpack/webpack-cli: README.md

## Supported arguments and commands

> Get to know about: what are the available commands and arguments [here](./packages/webpack-cli/README.md).

Reviewer: ```suggestion
All the available commands and arguments are [here](./packages/webpack-cli/README.md).
```

Author: OK sir,  let me fix this

### Utilities

> The project also have several utility packages which are used by other commands

Reviewer: prev is correct

Reviewer 2: Yes, `has` is correct.
![Screenshot at 2020-08-28 18-26-33](https://user-images.githubusercontent.com/46647141/91563186-14982600-e95c-11ea-96d3-d75faefa8445.png)
